game-core/src/main/java/games/strategy/ui/ImageScrollModel.java

```diff
@@ -1,14 +1,15 @@
 package games.strategy.ui;
 
-import java.util.Observable;
+import java.util.ArrayList;
+import java.util.Collection;
 
 /**
  * Model for an ImageScroller. Generally one large view and one small view will be connected to the
  * same model.
  *
  * <p>notifies its observers when changes occur.
  */
-public class ImageScrollModel extends Observable {
+public class ImageScrollModel {
   @SuppressWarnings("checkstyle:MemberName")
   private int x;
```

Comment:
I wonder if this suppression is still required?
This class doesn't look like it's actually implementing serializeable, maybe subclasses are doing it?
Anyways we should be able to remove it anyways?

Reply:
Still needed, I would caution though about comments in PRs that are on existing code or areas that are outside of the intended update, there are SO many things that could be updated. 
 
![Screenshot from 2019-09-30 11-36-42](https://user-images.githubusercontent.com/12397753/65906044-995a7880-e376-11e9-88d4-3bb4df8196f8.png)

On another note, it's very notable that the listener relies on the listeners to have references to this class and/or are just relying on side effects. It's pretty dirty.. Ideally a listener is given input of what has changed as opposed to the tangle web of dependencies that we have :(

meanwhile, the observer deprecation warnings are clogging up the compiler output, makes it difficult to find actual errors or new problems, I feel like this is an easy enough type of fix to chip away at.